crates/core/src/host/host_controller.rs

```diff
@@ -240,7 +249,6 @@ impl HostController {
             energy_monitor: self.energy_monitor.clone(),
         };
         let module_host = spawn_rayon(move || Self::make_module_host(mhc.host_type, mcc)).await?;
-        module_host.start();
```

Comment:
This was not necessary anymore - we always start the module immediately now, so there's no point in having the start functionality anymore.

Reply:
So why are we starting it immediately now?